### ISSUE TITLE
Examples in documentation aren't showing up properly

### ISSUE THREAD

**Issue by eliasdorneles** [paver/paver#166]
See: http://paver.readthedocs.io/en/latest/getting_started.html

![selection_117](https://cloud.githubusercontent.com/assets/37565/15982800/7caae794-2f68-11e6-8b15-f71095fbd85b.png)

**Comment by Almad** [comment]
Good point. I only set up RTD in automatic way...I'll see whether it's going to support tweaking we are doing :(

**Comment by Almad** [comment]
I'd appreciate help with this; #167 is WIP, but still not working properly.

Source seems to be cogged according to logs, but when RTD runs their builder over it, it a) bails over paver not being importable, b) probably somehow overrides the cogged source version.

Not sure why.

**Comment by rcoup** [comment]
Can we fix the link in the project title on Github to at least point to the existing docs at https://pythonhosted.org/Paver/ ? Currently you get an almost-empty template at RTD.

**Comment by Almad** [comment]
Done. It would be good to fix the RTD though :(
